tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/statements/arrays/ArrayTest.java

```diff
@@ -132,4 +135,47 @@ public void testConnectorNegativeCases() {
         BAssertUtil.validateError(resultNegative, 0, "function invocation on type 'int[]' is not supported", 3, 18);
         BAssertUtil.validateError(resultNegative, 1, "function invocation on type 'string[]' is not supported", 8, 21);
     }
+
+    @Test(description = "Test arrays of types without implicit initial values")
+    public void testArrayImplicitInitialValues() {
+        Assert.assertEquals(arrayImplicitInitialValueNegative.getErrorCount(), 8);
+        BAssertUtil.validateError(arrayImplicitInitialValueNegative, 0, "array element type 'ObjInitWithParam' " +
+                "does not have an implicit initial value, use 'ObjInitWithParam?'", 53, 1);
+        BAssertUtil.validateError(arrayImplicitInitialValueNegative, 1, "array element type 'FT' does not have an " +
+                "implicit initial value, use 'FT?'", 74, 1);
+        BAssertUtil.validateError(arrayImplicitInitialValueNegative, 2, "array element type 'FTUnion' does not have" +
+                " an implicit initial value, use 'FTUnion?'", 89, 1);
+        BAssertUtil.validateError(arrayImplicitInitialValueNegative, 3, "array element type 'FTSingle' does not have" +
+                " an implicit initial value, use 'FTSingle?'", 92, 1);
+        BAssertUtil.validateError(arrayImplicitInitialValueNegative, 4, "array element type 'FTNUnion' does not have" +
+                " an implicit initial value, use 'FTNUnion?'", 95, 1);
+        BAssertUtil.validateError(arrayImplicitInitialValueNegative, 5, "array element type 'error<>' does not have" +
+                " an implicit initial value, use 'error<>?'", 100, 1);
+        BAssertUtil.validateError(arrayImplicitInitialValueNegative, 6, "array element type 'UN' does not have" +
+                " an implicit initial value, use 'UN?'", 115, 1);
+        BAssertUtil.validateError(arrayImplicitInitialValueNegative, 7, "array element type 'error<>' does not have" +
+                " an implicit initial value, use 'error<>?'", 141, 1);
+    }
+
+    @Test(description = "Test arrays of types without implicit initial values")
+    public void testArrayImplicitInitialValuesOfFiniteType() {
+        CompileResult negResult = BCompileUtil.compile(
+                "test-src/statements/arrays/array-implicit-initial-value-finite-type-negative.bal");
+        Assert.assertEquals(negResult.getErrorCount(), 4);
+        BAssertUtil.validateError(negResult, 0,
+                "array element type 'allInitNonZero' does not have an implicit initial value, use 'allInitNonZero?'",
+                22, 1);
+
```

Comment:
Shall we remove this extra line? Shall we do the same for all the extra new lines in this file?